app-utils/qtscrcpy/autobuild/prepare

```diff
@@ -0,0 +1,2 @@
+abinfo "Updating submodules..."
```

Comment:
So I know this was done to override that SSH repo - so please add a comment here and in the spec file, explaining why you've handled it manually in prepare.